travel/filters.py

```diff
@@ -0,0 +1,17 @@
+import django_filters
+from .models import Trip, Event, Region, EventType
+from cities_light.models import Country
+
+class TripFilter(django_filters.FilterSet):
+    Principal_title = django_filters.CharFilter(lookup_expr='icontains', name='principal__title')
```

Comment:
Is there a reason for these attributes to be Upper_Case_with_Underscores? Can we just make them lower_case?